core/src/main/java/com/linecorp/armeria/common/logging/DefaultRequestLog.java

```diff
@@ -90,22 +74,24 @@
     private static final ResponseHeaders DUMMY_RESPONSE_HEADERS = ResponseHeaders.of(HttpStatus.UNKNOWN);
 
     private final RequestContext ctx;
+    private final CompleteRequestLog notCheckingAccessor = new CompleteRequestLog();
```

Comment:
By the way this was a brainstorming proposal - it seems good on paper, but we might want to benchmark before adding this class.

Reply:
Bimorphic calls are almost as fast as monomorphic calls, so I guess it will be a win, although we allocate one extra object?

Reply:
Yeah it's my guess too, but generally better to benchmark than guess ;) But it's fine as is for now either way.